test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/ProjectSystems/LegacyPackageReferenceProjectTests.cs

```diff
@@ -805,6 +805,49 @@ public async Task GetPackageSpecsAsync_SkipContentFilesAlwaysTrue()
             }
         }
 
+        [Fact]
+        public async Task GetPackageSpecAsync_CentralPackageVersionsValidation()
+        {
+            // Arrange
+            var packageA = (PackageId: "packageA", Version: "1.2.3");
+            var packageB = (PackageId: "packageB", Version: "3.4.5");
+            var packageAA = (PackageId: "packageA", Version: "5.0.0");
```

Comment:
I think `packageAv1` and `packageAv5` would be easier to understand. `packageAA` makes it sound like it's a different package to `packageA`.

Reply:
Changed the name.